static/app/views/issueDetails/streamline/header/header.tsx

```diff
@@ -156,12 +157,19 @@ export default function StreamlinedGroupHeader({
                 </StatLink>
               ))}
           </StatTitle>
-          <EventMessage
-            data={group}
-            level={group.level}
-            message={secondaryTitle}
-            type={group.type}
-          />
+          <Tooltip
+            title={t(
+              'The main message summarizing the event. Typically the error message or log string that triggered this issue.'
+            )}
+            isHoverable
+          >
```

Comment:
use skipWrapper where possible which basically just doesn't render an extra span. If you're wrapping a div or an existing span you don't need another span.
```suggestion
          <Tooltip
            title={t(
              'The main message summarizing the event. Typically the error message or log string that triggered this issue.'
            )}
            isHoverable
            skipWrapper
          >
```